<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Save_Form</class>
 <widget class="QWidget" name="Save_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранить</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="CatalogPath">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="SelectCatalog">
     <property name="text">
      <string>Выбрать каталог</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Формат кадра:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FormatFrame"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Размер сохраняемого кадра:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SizeFrame"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
